### PR TITLE
Ghost should always spawn when players spawn 

### PR DESCRIPTION
**Pull request by Bod9001**
### Purpose
starting work towards mind rework

### CODE REVIEW

**Comment by ThatDan123** [comment]
Codacy stuff

File: UnityProject/Assets/Scripts/Player/GhostMove.cs [hunk 1]
```diff
@@ -13,11 +13,11 @@ public class GhostMove : NetworkBehaviour, IPlayerControllable
 
 	public bool Moving = false;
 
-	private RegisterTile registerTile;
+	public RegisterTile registerTile;
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Player/GhostMove.cs]:
```suggestion
	public RegisterTile RegisterTile {get; private set;};
```

Can you do that for all of these

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Player/GhostMove.cs]: I was  Settingit through the editor

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Player/GhostMove.cs]:
```suggestion
	public RegisterTile RegisterTile {get; set;};
```
Do that then, it just doesnt need to be shown in the inspector.

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Player/GhostMove.cs]: oh ok, i cant read

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Player/GhostMove.cs]: Cant you just [SerialiseField] then? I don't see why they need to be public?

**Comment by corp-0** [comment]
Try this
[field: SerializeField]
public int Value
{
    get;
    private set;
}

File: UnityProject/Assets/Scripts/Player/GhostMove.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Player/GhostMove.cs]: ok

**Comment by Bod9001** [comment]
does that work of the unity editor I thought it was incapable of doing getters and setters?

**Comment by Bod9001** [comment]
'GhostMove.registerTile.set' accessor must be more restrictive than the property or indexer 'GhostMove.registerTile'

File: UnityProject/Assets/Scripts/Player/GhostMove.cs [hunk 2]
```diff
@@ -13,11 +13,14 @@ public class GhostMove : NetworkBehaviour, IPlayerControllable
 
 	public bool Moving = false;
 
-	public RegisterTile registerTile;
+	[SerializeField] private RegisterTile registerTile {
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Player/GhostMove.cs]: Would need to be public to get rid of that error

**Pull request merged**